<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>Aibek Kappassov's webpage</title>

    <link rel="stylesheet" href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/pygment_trac.css">
    <!-- Add icons -->
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
    <meta name="viewport" content="width=device-width">
    <!-- Google tag (gtag.js) -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=G-FDTCR0VGNB"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());
    
      gtag('config', 'G-FDTCR0VGNB');
    </script>
    <!--[if lt IE 9]>
    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
  </head>
  <header>
        <h1 align="center">Aibek Kappassov</h1>

        <!-- Picture + Links + News -->
        <div style="margin-bottom: 10px" align="center">


            <img src="./images/aibek.JPG" alt="Portrait" class="portrait">
            <!-- Portrait -->
            <!-- <img src="images/email.png" alt="Portrait" WIDTH=280 style="margin-bottom: 8px;border-radius: 10px;"> -->
            <p> Contact: <a href="mailto:akappassov@gmail.com">akappassov [at] gmail.com</a> </p>
            <p>

            <!-- Email -->
            <!-- <div style="margin-bottom: 6px;"><img src="./files/email.png" width="175"></div> -->
            <!-- <p> <a>xxxxxx@ugr.es</a> | <a>xxxxxx@go.ugr.es</a></p> -->

            <!-- CV -->
            <a href="./docs/AK_CV.pdf" style="margin-right:14px;"><i class="ai ai-cv" style="font-size: 32px;"></i>
            </a>

            <!-- Git -->
            <a href="https://github.com/aibek22" style="margin-right:14px;">
                <i class="fa fa-github" style="font-size:32px;"></i>
                <!-- color: #A7A499" -->
            </a>

            <a href="https://www.linkedin.com/in/aibekkapp" style="margin-right:14px;">
                <i class="fa fa-linkedin" style="font-size:32px;"></i>
                <!-- color: #A7A499" -->
            </a>


            <!-- Scholar -->
            <a href="https://scholar.google.ca/citations?user=F-CRoUEAAAAJ&hl=en&oi=ao" style="margin-right:14px;">
                <i class="ai ai-google-scholar big-icon" style="font-size:32px;"></i>
            </a>

            <br>
            <br>
        </div>

  </header>

  <section>

    <!-- <h1> <a>Website in progress ..... </a> &#9935;</h1> -->



    <h2> About me</h2>

    <p> I am a computational biologist with a background in mathematics (Purdue University) and research interests in self-organization in living matter and developmental dynamics. Previously, I was at the <a href="https://www.imperial.ac.uk/integrative-systems-biology/about/">Centre for Integrative Systems Biology and Bioinformatics (CISBIO)</a> at Imperial College where I wrote my Master's thesis with <a href="https://profiles.imperial.ac.uk/r.endres">Prof. Robert Endres</a>. </p>




    <h2> Recent news</h2>

    <ul>
     <!-- <li> <a>February 2025</a> Joining the <a href="https://sites.google.com/view/shinlab/home?authuser=0">Shin Lab</a> at Seoul National University as a research intern in Spring '25.</li> -->

      <li> <a>January 2025</a> Endres Group's new <a href="https://www.nature.com/articles/s41598-025-86854-7">paper</a> on Turing patterns is now out!</li>
   
    </ul>


    <h2> Projects </h2>

    <table style="margin-top:-12px;">

                    <!-- Work -->
                    <tbody><tr style="border-bottom:1px solid #e5e5e5;">
                        <!-- IMAGE -->
                        <td>
                            <img class="workPicture" src="images/turing.jpg" alt="Turing Pattern">
                        </td>
        
                        <!-- TITLE AND INFO -->
                        <td>
                            <br>
                            <div id="indexWorkText">
                                <!-- Title + Link -->
                                <b>Optimal Network Sizes for Most Robust Turing Patterns

                                </b>
        
                                <br>
                                <i>Scientific Reports 2025</i>
                            </div>
                            <div class="indexWorkText">
                                <a href="https://www.nature.com/articles/s41598-025-86854-7" class="button" target="_blank">PAPER</a>
                                <a href="https://github.com/Endres-group/Random_Turing_networks" class="button" target="_blank">CODE</a>
                                <!-- <a href="https://huggingface.co/datasets/tomg-group-umd/cinepile" class="button" target="_blank">DATASET</a> -->
                                <!-- <a href="" class="button" target="_blank">VIDEO</a> -->
                              </div>
                            <br>
                            <!-- One sentence description -->
                            <div id="indexWorkText">
                                Many cellular patterns exhibit a reaction-diffusion component, suggesting that Turing instability may contribute to pattern formation. However, biological gene-regulatory pathways are more complex than simple Turing activator-inhibitor models and generally do not require fine-tuning of parameters as dictated by the Turing conditions. In this paper, we employ random matrix theory to analyze the Jacobian matrices of larger networks with robust statistical properties. Our analysis reveals that Turing patterns are more likely to occur by chance than previously thought and that the most robust Turing networks have an optimal size, consisting of only a handful of molecular species, thus significantly increasing their identifiability in biological systems.
                            </div>
                            <br>
        
                            <!-- Collaborators -->
                            <div id="indexWorkText">
                                H. Shaberi, A. Kappassov, A. Matas-Gil, R. G. Endres
                            </div>
                            <br>
                            <br>
                        </td>
                    </tr>
                    <!-- Work -->
                    <tbody><tr style="border-bottom:1px solid #e5e5e5;">
                        <!-- IMAGE -->
                        <td>
                            <img class="workPicture" src="images/epistasis.jpg" alt="Epistasis">
                        </td>
        
                        <!-- TITLE AND INFO -->
                        <td>
                            <br>
                            <div id="indexWorkText">
                                <!-- Title + Link -->
                                <b>A Mathematical Framework for Describing and Predicting Epistasis in a Synthetic Gene Circuit

                                </b>
        
                                <br>
                  
                            </div>
                            <div class="indexWorkText">
                                <a href="https://github.com/aibek22/Epistasis_Project_MSc" class="button" target="_blank">CODE</a>
                    
                                <!-- <a href="" class="button" target="_blank">VIDEO</a> -->
                              </div>
                            <br>
                            <!-- One sentence description -->
                            <div id="indexWorkText">
                                Epistasis is the phenomenon in which the effect of genotype on phenotype is dependent on a combination of two or more mutations. It plays a significant role in contributing to the phenotypic differences between and within species. Studying epistasis can have various therapeutic applications. In this project, we try to predict epistasis and its effect a priori using mathematical modelling. We present epistasis predicted by our models and discuss their performance. Finally, we talk about alternative inference techniques that can be applied in future studies.
                            </div>
                            <br>
        
                            <!-- Collaborators -->
                            <div id="indexWorkText">
                                Supervisor: Dr. Ruben Perez-Carrasco
                              <br>
                                Contributors: Aibek Kappassov, Aaryan Bhatia, Robin Curnow, Chi Chun Chan
                            </div>
                            <br>
                            <br>
                        </td>
                    </tr>

                    </tbody></table>


    <h2> Publications </h2>
    <p> <a href="https://www.nature.com/articles/s41598-025-86854-7">Optimal network sizes for most robust Turing patterns.</a> H. Shaberi, <strong>A. Kappassov</strong>, A. Matas-Gil, R. G. Endres. <em><strong>Scientific Reports</strong></em> (2025) </p>


      
    <h2> Awards </h2>
    <ul>
      <li> <a>April 2020</a> Thomas Arai Scholarship <a href="https://www.math.purdue.edu/files/purview/PUrview2020.pdf">Department of Mathematics</a>, Purdue University.

    </ul>
<!-- fellowship -->

  </section>

      <footer>
        <p><small>Theme by <a href="https://github.com/orderedlist">orderedlist</a> and <a href="https://github.com/david-abel/minimal">David Abel</a></small>  </p>
      </footer>
    </div>
    <script src="javascripts/scale.fix.js"></script>

</html>
